tools/releases/copy-image-artifacts.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+# this script copies artifacts from a .dockeringore file to a build folder
+
+if [[ $# -ne 3 ]]; then
+  usage
+  exit 1
+fi
+
+function usage() {
+  echo "Usage: $0 <component> <os> <arch>"
+}
+
+DOCKERIGNORE_FILE="tools/releases/dockerfiles/Dockerfile.$1.dockerignore"
+GOOS="$2"
+GOARCH="$3"
+ARTIFACTS_DIR="build/artifacts-$GOOS-$GOARCH"
+DESTINATION_DIR="$ARTIFACTS_DIR/$1"
+
+function safe_cp() {
+  FILE="$1"
+  DIR_1=$(dirname "$FILE")
+  DEST="$2"
+  FINAL_DEST="$DEST/$DIR_1/"
+  # shellcheck disable=SC2086
+  mkdir -p "$FINAL_DEST" && cp -R $FILE "$FINAL_DEST"
```

Comment:
2. This can be a glob like `*` - `build/artifacts-linux-arm64/envoy/*` so escaping it will not expand the glob and not copy all the files. I will change it to a for loop.

Reply:
done